src/index.html

```diff
@@ -6,8 +6,38 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700;800&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="product-card">
+      <img
+        class="product-card__image"
+        src="images/imac.jpeg"
+        alt="123">
```

Comment:
and logo should be wrapped with a link
```suggestion
      <img
        class="product-card__image"
        src="images/imac.jpeg"
        alt="123"
      >
```